client/modules/defaultView.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from '../css/default.module.css';
+import TotalScore from './totalScore.jsx';
+import Categories from './Categories.jsx';
+
+class DefaultView extends React.Component {
```

Comment:
This is a good idea. I'm also thinking about implementing a default view for my component.